src/modules/reportes_poai/reportes_poai.service.ts

```
@@ -0,0 +1,665 @@
+        },
+      };
+    } catch (error) {
+      return {
+        status: false,
+        message: 'Error al obtener historiales',
+        error: error.message,
+      };
+    }
+  }
+
+    async generarReportePoai(historialId: number): Promise<Buffer> {
```

ESLint

[error] Delete `··`

(prettier/prettier)